tests/helper.py

```diff
@@ -8,26 +8,30 @@
 from requests.exceptions import HTTPError
 
 
-class ResponseMock:
+class ResponseMock:  # pylint: disable=too-many-instance-attributes
     """This class mocks a http response object."""
 
     def __init__(self, **kwargs: Any) -> None:
         """
-        Give the mock the response you except.
-
-        Args:
-            text (str, optional): text response. Defaults to "".
-            json (Any, optional): json response. Defaults to "".
-            status (int, optional): http status code. Defaults to 444 (No Response)
+        Give the mock the response you expect.
+
+        **kwargs:
+            content: (Any, optional): Content of the response
+            headers: (List, optional): List of headers
+            json (Any, optional): JSON response. Defaults to ""
+            ok (bool, optional):  The OK flag. Defaults to True
+            reason(str, optional): The response reason. Defaults to ""
+            status (int, optional): HTTP status code. Defaults to 444 (No Response)
```

Comment:
Adapt here `status` to `status_code`, perhaps you want to add the "known" kwargs to the __init__(...) declaration too?

Reply:
Thank you, I missed that.